web/src/main/java/com/epam/esm/web/config/WebConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.web.config;
+
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.epam.esm")
+public class WebConfig implements WebMvcConfigurer {
+
+   /* private final ServiceConfig serviceConfig;
```

Comment:
there should be only necessary comments in repos

Reply:
@valevaty98 done